source/ssb-chart/README.md

```diff
@@ -1,3 +1,26 @@
+# ssb-chart v2.2.0
+
+Support for not running cloudsql-proxy and revert the change introduced in v2.1.0
```

Comment:
Would it be an idea of highlighting the new feature in the beginning of the message. E.g something along the lines of:

> Support for using Cloud SQL connectors instead of the CloudSql proxy.
> Cloud SQL connectors support IamAuthentication which gives improved security over cloudSql proxy.
> If you're using Go, Java, or Python, consider using the corresponding Cloud SQL connector which does everything the proxy does.
> 
> https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/main/README.md


Or something like that. :)

Reply:
Good point! I've improved based on your suggestion